pkgs/applications/editors/jetbrains/default.nix

```diff
@@ -127,9 +127,9 @@ let
     }).overrideAttrs (attrs: {
       postFixup = (attrs.postFixup or "") + lib.optionalString stdenv.isLinux ''
         interp="$(cat $NIX_CC/nix-support/dynamic-linker)"
-        patchelf --set-interpreter $interp $out/goland*/plugins/go/lib/dlv/linux/dlv
+        patchelf --set-interpreter $interp $out/goland*/plugins/go-plugin/lib/dlv/linux*/dlv
 
-        chmod +x $out/goland*/plugins/go/lib/dlv/linux/dlv
+        chmod +x $out/goland*/plugins/go-plugin/lib/dlv/*/dlv
```

Comment:
Does the current packaging work for linux aarch64?

Reply:
assuming that linux aarch64 works, should we patch just the relevant binary for the platform?

Reply:
To be honest, I don't know if linux aarch64 works. But I trusted patchelf to do the right thing.. My guess was that if `linux/dlv` needed to be patched, than `linux-aarch64/dlv` needs to be patched as well. But I might be wrong.

Reply:
I changed it now to patch only the relevant binary...